src/Box.js

```diff
@@ -4,11 +4,38 @@ import PropTypes from 'prop-types'
 
 const align = responsiveStyle('text-align', 'align')
 
+
+function boxShadow(props) {
+  const shadowColor = 'rgba(0,0,0,0.08)'
+  const shadows = [
+    `0 0 4px 0 ${shadowColor}`, 
+    `0 8px 8px 0 ${shadowColor}`, 
+    `0 16px 16px 0 ${shadowColor}`, 
+    `0 32px 32px 0 ${shadowColor}`
+  ]
+  const boxShadows = {
+    'small': {
+      'box-shadow': shadows[0]
+    },
+    'medium': {
+      'box-shadow': shadows.slice(0,2).join(", ")
+    },
+    'large': {
+      'box-shadow': shadows.slice(0,3).join(", ")
+    },
+    'xlarge': {
+      'box-shadow': shadows.join(", ")
+    }
+  }
```

Comment:
Maybe I'm missing something here... why include multiple entries from the `shadows` array in the `box-shadow` value for larger sizes?

Reply:
@747823: I followed the box-shadow styles that @odmln provided https://github.com/pricelinelabs/design-system/issues/65